Lib/pdb.py

```diff
@@ -586,10 +586,10 @@ def _complete_expression(self, text, line, begidx, endidx):
     # Return true to exit from the command loop
 
     def do_commands(self, arg):
-        """commands [bpnumber]
-        (com) ...
-        (com) end
-        (Pdb)
+        """(Pdb) commands [bpnumber]
+           (com) ...
+           (com) end
+           (Pdb)
```

Comment:
Just curious, why the indent?

Reply:
Just to align better in docstring. This is what the users see when they use this command.

Reply:
Hm, I sort of prefer it as is. It doesn't affect how pdb's help is formatted, but it does break help formatting for this function itself:

`help(pdb.Pdb.do_commands)` before:

```
Help on function do_commands in module pdb:

do_commands(self, arg)
    commands [bpnumber]
    (com) ...
    (com) end
    (Pdb)

    Specify a list of commands for breakpoint number bpnumber.
```

After:

```
Help on function do_commands in module pdb:

do_commands(self, arg)
    (Pdb) commands [bpnumber]
       (com) ...
       (com) end
       (Pdb)

    Specify a list of commands for breakpoint number bpnumber.
```